### PR TITLE
Doc Update  - whats new v1.0.2 + TimeSeriesByCategory

### PR DESCRIPTION
**Pull request by mail4umar**
- Added new page for whats new v1.0.2 in the doc
- Updated TimeSeries RST page to include the new class: TimeSeriesByCategory
- Updated docstring for the class: TimeSeriesByCategory

### CODE REVIEW

File: docs/source/whats_new_v1_0_2.rst [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+Time Series
+------------
+
+We added a new Time Series class: ``TimeSeriesByCategory``. This allows the users to build multiple models based off on a category. The number of models created
+are equal to the categories. This saves users time to create multiple models separately. For more inofrmation please see :py:func:`~verticapy.machine_learning.vertica.tsa.ensemble.TimeSeriesByCategory`.
+
+Plots
+--------------
+
+ - Two new plots have been added for plotly that were previously missing: Scree + Var.
```
Comment (oualib) [inline docs/source/whats_new_v1_0_2.rst:42]: please add ref to the plots, is there a chart gallery page for them?

File: docs/source/whats_new_v1_0_2.rst [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+- Additioanlly we have added a ``prompt`` option for :py:func:`~verticapy.connection.new_connection`. This allows the user to enter the secrets discretly with a masked display.
+
+Time Series
+------------
+
+We added a new Time Series class: ``TimeSeriesByCategory``. This allows the users to build multiple models based off on a category. The number of models created
+are equal to the categories. This saves users time to create multiple models separately. For more inofrmation please see :py:func:`~verticapy.machine_learning.vertica.tsa.ensemble.TimeSeriesByCategory`.
+
+Plots
+--------------
```
Comment (oualib) [inline docs/source/whats_new_v1_0_2.rst:40]: why so many '-'

File: docs/source/machine_learning_vertica_time_series.rst [hunk 3]
```diff
@@ -4,6 +4,60 @@
 Time Series
 ===============
 
+
+Multi-Timeseries Model
+~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (oualib) [inline docs/source/machine_learning_vertica_time_series.rst:9]: write BETA

File: docs/source/whats_new_v1_0_2.rst [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+These updates significantly enhance the accessibility, debugging, and enhancement capabilities of your queries.
+
+OAuth Refresh Tokens
+---------------------
+
+- We have updated the connector to accept OAuth refresh tokens.
+- Additioanlly we have added a ``prompt`` option for :py:func:`~verticapy.connection.new_connection`. This allows the user to enter the secrets discretly with a masked display.
+
+Time Series
```
Comment (oualib) [inline docs/source/whats_new_v1_0_2.rst:33]: write BETA

**Review by oualib — CHANGES_REQUESTED** [review]
Please correct the little changes + change the version flag to 1.0.2 in the init and in the setup.

File: docs/source/whats_new_v1_0_2.rst (hunk 1 above)
Comment (mail4umar) [reply docs/source/whats_new_v1_0_2.rst]: We don't have them yet. But we can create the page in next iteration.

**Pull request merged**